src/core/utils/EventNormalizer.ts

```diff
@@ -918,8 +918,8 @@ export class EventNormalizer {
             const target = this._mousedownInEditable.target as Element;
             this._mousedownInEditable = null;
             if (ev.target instanceof Element) {
-                let range: Range;
-                if (target === ev.target) {
+                let range: Range = this._getRange();
+                if (!target.contains(range.startContainer) && target === ev.target) {
```

Comment:
This particular part of the EventNormalizer is lacking documentation, but it's not your fault, it's simply something I notice now.